Agent/Linux/ubuntu1804-README.md

```diff
@@ -0,0 +1,44 @@
+# Azure DevOps - Ubuntu 18.04 Agent
+
+The following sections provide a step-by-step guide to creating a Azure DevOps Agent Image using Packer, spin up an Agent using the Image and add it to the Azure DevOps Agent Pool.
+
+## Prepare your Azure DevOps Account
+
+1. Create an Azure DevOps Agent pool.
+
+2. Generate a Personal Access Token (PAT) for your Azure DevOps Organization. When generating the Personal Access Token (PAT), assign the following scopes:
```

Comment:
Same as Number 1, let's find an existing step by step guide from Azure DevOps doc site.

Reply:
Added a link to the documentation for creating an Azure DevOps Agent pool.